code/modules/mob/living/simple_animal/hostile/terror_spiders/ghost.dm

```diff
@@ -16,6 +16,8 @@
 	humanize_prompt += " Role: [spider_role_summary]"
 	if(user.ckey in ts_ckey_blacklist)
 		error_on_humanize = "You are not able to control any terror spider this round."
+	else if(user.has_enabled_antagHUD && config.antag_hud_restricted)
```

Comment:
`has_enabled_antagHUD == 1`

This can actually be `2` under some circumstances.

Reply:
👍